Lsunae/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/view/product/base/BaseActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.exam.elevenstreet.view.product.base
+
+import android.os.Bundle
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+
+abstract class BaseActivity<T : ViewDataBinding>(@LayoutRes private val layoutId: Int) :
+    AppCompatActivity() {
+    protected lateinit var activityMainBinding: T
```

Comment:
```suggestion
    protected lateinit var binding: T
```
base가되는 activity의 binding 객체이름에 'main' 이라는 이름을 넣으면 안될 것 같아요.

Reply:
수정했습니다!
acc6b7d